pkg/config/analysis/msg/messages.yaml

```diff
@@ -50,6 +50,7 @@ messages:
       - name: podName
         type: string
 
+# Deprecated: Correcting refactoring to gateway port is not on workload
```

Comment:
Is this message being used elsewhere? If not, can you refer to messages like `# IST0114 RETIRED`? There should be examples on how to deprecate the message.

Reply:
There's no other use. I'm going to learn 0114

Reply:
Thanks, I think the deprecated message description in istio.io page should also be deleted. Otherwise it will cause doc test failing.

Reply:
`# IST0114 RETIRED` has been referred to.

@hanxiaop  PTAL, thanks